.github/workflows/links.yml

```
@@ -1,45 +1,37 @@
+          fail: true
+          failIfEmpty: true
+          debug: false
 
-      - name: Set lychee exit code
-        run: echo "lychee_exit_code=$?" >> $GITHUB_ENV
-
-      - name: Save lychee cache
-        uses: actions/cache/save@v4
+      - name: Upload lychee output
         if: always()
+        uses: actions/upload-artifact@v2
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)